lib/posix/posix.nim

```diff
@@ -1566,6 +1566,8 @@ var
     ## Receive timeout.
   SO_REUSEADDR* {.importc, header: "<sys/socket.h>".}: cint
     ## Reuse of local addresses is supported.
+  SO_REUSEPORT* {.importc, header: "<sys/socket.h>".}: cint
```

Comment:
I guess we're going to need to introduce a ``enableReusePort`` define. Unless somebody has a better idea?